synth.py

```diff
@@ -93,16 +93,19 @@
 templated_files = common.py_library(
     samples=True,  # set to True only if there are samples
     microgenerator=True,
+    unit_test_extras=["pandas"],
+    system_test_extras=["pandas"],
     cov_level=99
 )
 s.move(templated_files, excludes=[".coveragerc"])  # microgenerator has a good .coveragerc file
 
+# Don't treat warnings as errors.
```

Comment:
My initial thought is we should treat warnings as errors so that maintainers can be notified in case of new warnings. I wasn't able to see any existing warnings when running `synthtool`. If there are warnings we could create a separate issue in this repo to track the work item of going through and either addressing the warnings or suppressing them individually.

Reply:
Ah thanks for trying that out. I'll remove it from this synth.py.

Reply:
I uncommented this and it broke the the `docs` session. The comment is not about library test warnings, just about docs warnings. I clarified that in the comment and opened https://github.com/googleapis/python-monitoring/issues/95 to track digging into why it's needed